Compile error in `bevy_core_pipeline` when `dds` feature is enabled

## Bevy version

`0.13`, `efda05d11e1bc8f833294011d3f17e659a4f8f4e`

## What you did

Run `cargo check -p bevy_core_pipeline -F dds` in the bevy repo, or have `bevy_core_pipeline` with `dds` feature enabled as a dependency in your app/library code.

## What went wrong

I get the following error:

```
    Checking bevy_core_pipeline v0.13.0 (/home/tim/dev/bevyengine/bevy/crates/bevy_core_pipeline)
error[E0061]: this function takes 6 arguments but 7 arguments were supplied
   --> crates/bevy_core_pipeline/src/tonemapping/mod.rs:357:5
    |
357 |       Image::from_buffer(
    |       ^^^^^^^^^^^^^^^^^^
358 |           #[cfg(all(debug_assertions, feature = "dds"))]
359 |           "Tonemapping LUT sampler".to_string(),
    |           -------------------------------------
    |           |
    |  _________unexpected argument of type `std::string::String`
    | |
360 | |         bytes,
    | |________- help: remove the extra argument
    |
note: associated function defined here
   --> /home/tim/dev/bevyengine/bevy/crates/bevy_render/src/texture/image.rs:645:12
    |
645 |     pub fn from_buffer(
    |            ^^^^^^^^^^^

For more information about this error, try `rustc --explain E0061`.
error: could not compile `bevy_core_pipeline` (lib) due to 1 previous error
```

It looks like the `dds` feature of `bevy_mod_core_pipeline` must enable the `dds` feature of `bevy_render` to fix this.

## Additional information

This is breaking the CI pipeline of my plugin: <https://github.com/TimJentzsch/bevy_mod_krita/actions/runs/7951590457/job/21705198887?pr=23#step:6:177>

--- a/crates/bevy_core_pipeline/Cargo.toml
+++ b/crates/bevy_core_pipeline/Cargo.toml
@@ -13,7 +13,7 @@ license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
 [features]
-dds = []
+dds = ["bevy_render/dds"]
 trace = []
 webgl = []
 webgpu = []

